src/Libraries/Microsoft.Extensions.ObjectPool.DependencyInjection/PoolOptions.cs

```diff
@@ -1,19 +1,23 @@
-﻿// Licensed to the .NET Foundation under one or more agreements.
+// Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
 using System.Diagnostics.CodeAnalysis;
 
-namespace Microsoft.Extensions.DependencyInjection.Pools;
+namespace Microsoft.Extensions.ObjectPool;
 
 /// <summary>
 /// Contains configuration for pools.
 /// </summary>
 [Experimental]
 public sealed class PoolOptions
```

Comment:
Given that this is now in the general-purpose Microsoft.Extensions.ObjectPool namespace, perhaps we should give this a more specialized name? ServiceCollectionPoolOptions maybe? Any other ideas?

Reply:
This one is hard, `DependencyInjectionPoolOptions`? But the current name loks fine to me, there is no conflict and users will probably not use the class name directly.

Reply:
Why didn't this get called ObjectPoolOptions?

Reply:
I agree with Martin. It looks strange, that this type (or ObjectPoolOptions) is not used by `Microsoft.Extensions.ObjectPool` || `Microsoft.Extensions.ObjectPool<T>`.